cpp/ql/src/Architecture/General Namespace-Level Information/GlobalNamespaceClasses.ql

```diff
@@ -1,7 +1,8 @@
 /**
  * @name Global namespace classes
  * @description Finds classes that belong to no namespace.
- * @kind table
+ * @kind problem
+ * @problem.severity recommendation
```

Comment:
Do we want a `@precision` tag as well?

Reply:
I'm a bit reluctant as I believe it's precise, but I don't want it picked up to run somewhere (like LGTM) as this is a somewhat specialized query.

Reply:
That's fair. I see now that we have plenty of other `@problem.severity recommendation` queries without a `@precision` tag so it's probably fine to introduce another one.